content/ADR-189-social-service-M2.md

```diff
@@ -0,0 +1,165 @@
+
+## Risk analysis
+
+Description of risk — Summary description of the risk—easy to understand.
+
+Probability of occurrence — Estimate the probability that this risk will materialize (HIGH-MEDIUM-LOW).
+
+Severity — The intensity of undesirable impact on the project—if the risk materializes (HIGH-MEDIUM-LOW)..
+
+Action — The contingent response if the risk materializes.
+
+Loss Size — Given in hours or days, this is a measure of the negative impact on the project.
+
+Risk Exposure — Given in hours or days, this is a product of probability and loss size.
+
+Priority (optional) — This is either an independent ranking or the product of probability and severity. Typically, a higher-severity risk with high probability has higher relative priority (1-3) 1 being the higher.
+
+| Description of risk                                                                | Probability of occurrence                       | Severity | Mitigation                                                                                                                                                                    | Loss Size                                                                                                                                                   | Risk Exposure | Priority |
+| ---------------------------------------------------------------------------------- | ----------------------------------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | -------- |
+| Rust-RPC can’t handle the expected load                                            | Low                                             | High     | Invest time in developing the framework to make it more performant                                                                                                            | If this problem gets to prod, users might be impacted by not receiving all expected messages. Or in the worst case, take down the social service altogether | Medium        | 1        |
+| Rust RPC has internal bugs                                                         | Medium                                          | High     | Invest time in developing the framework to reduce bugs                                                                                                                        | If this problem gets to prod, users might be impacted by not receiving all expected messages. Or in the worst case, take down the social service altogether | High          | 1        |
```

Comment:
Choose either `Rust RPC` or `Rust-RPC` and use it consistent in all occurrences in this doc.